client/components/card/index.jsx

```diff
@@ -25,38 +37,26 @@ class Card extends Component {
 	};
 
 	render() {
-		const { children, compact, highlight, href, onClick, tagName, target } = this.props;
-
-		const highlightClass = highlight ? 'is-' + highlight : false;
-
-		const className = classnames(
-			'card',
-			this.props.className,
-			{
-				'is-card-link': !! href,
-				'is-clickable': !! onClick,
-				'is-compact': compact,
-				'is-highlight': highlightClass,
-			},
-			highlightClass
-		);
-
-		const omitProps = [ 'compact', 'highlight', 'tagName' ];
+		const {
+			children,
+			compact, // eslint-disable-line no-unused-vars
+			highlight, // eslint-disable-line no-unused-vars
+			highlightClass, // eslint-disable-line no-unused-vars
```

Comment:
I wonder if we can eventually make eslint a little smarter for this case. I ran into this before as well

Reply:
ESLint just got smarter a moment ago when #24908 got merged :wink: